tap_jira/streams.py

```diff
@@ -134,6 +147,9 @@ def sync_on_prem(self):
                 path = "/rest/api/2/project/{}/version".format(project["id"])
                 pager = Paginator(Context.client, order_by="sequence")
                 for page in pager.pages(VERSIONS.tap_stream_id, "GET", path):
+                    # Transform userReleaseDate and userStartDate values to 'yyyy-mm-dd' format.
+                    for each_page in page:
```

Comment:
Correct me if I'm wrong, but `each_page` here is actually a record. So can we rename this variable?